src/js/edu-benefits/actions/index.js

```diff
@@ -8,7 +8,7 @@ export const UPDATE_SUBMISSION_TIMESTAMP = 'UPDATE_SUBMISSION_TIMESTAMP';
 export const VETERAN_FIELD_UPDATE = 'VETERAN_FIELD_UPDATE';
 export const ENSURE_FIELDS_INITIALIZED = 'ENSURE_FIELDS_INITIALIZED';
 
-export function ensureFieldsInitialized(section) {
+export function ensureSectionInitialized(section) {
   return (dispatch, getState) => {
     return dispatch({
       type: ENSURE_FIELDS_INITIALIZED,
```

Comment:
No, but we could if this is confusing. `ensureSectionInitialized` does the same thing as ensureFieldsInitialized, except that it gets the list of fields based on the section name you pass to it. I did it so that I didn't have to pass `uiState.sections` to some components in order to dirty all the fields in a section.